.azure/azure-pipelines.ci.yml

```diff
@@ -203,6 +203,23 @@ stages:
 # Performance Tests
 #
 
+- stage: remoteperformance
```

Comment:
So practically, we should be replacing the performance templates & scripts with this, right? Let's remove all those and replace them.

Reply:
I'll need to make sure it still handles linux. If it does we can do that.

Reply:
Also, this doesn't have PGO in it yet. I can work on adding that.

Reply:
Remove the word "remote" from everything when you're done.